roles/backup/templates/management-pod.yml.j2

```diff
@@ -20,9 +20,33 @@ spec:
     resources:
       {{ backup_resource_requirements | to_nice_yaml(indent=2) | indent(width=6, first=False) }}
 {%- endif %}
-{% if db_management_pod_node_selector %}
+{% if task_node_selector %}
```

Comment:
@ranvit The `task_node_selector` parameter only exists in the installer role (AWX CR), not in the back role (AWXBackup CR).  

So db_management_pod_node_selector is correct.  

If you need topologySpreadConstraints, tolerations, and affinity as well, those params would need to be added to the AWXBackup CR and should be prefixed with `db_management_pod_*`

Reply:
This PR also needs a rebase now to resolve conflicts.